src/permutation.rs

```diff
@@ -96,6 +93,110 @@ impl Permutation {
         debug_assert!(result.valid());
         return result;
     }
+    /// Computes the permutation that would sort a given slice.
```

Comment:
I think the naming is a bit confusing, since `my_permutation.sort(vec)`  makes it sound like it's going to sort the vec.  

How about naming it `copy_from_sort` `assign_from_sort` or something like that?  I would also drop the copied docstring text since I think it is better to keep it short and emphasize that it is equivalent to permutations::sort() with assignment, with a comment that it saves an allocation.

The example code should use something other than ones(), to demonstrate the value.

Reply:
Agreed, I can fix those issues.